website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
+                sequences. The core team is at the <a href='https://bsse.ethz.ch/cevo'>cEVO group </a> at ETH Zurich, but
+                we are grateful for our many <a
+                    href='https://github.com/GenSpectrum/cov-spectrum-website?tab=readme-ov-file#contributors-'
+                >
+                    contributors
+                </a> from around the globe. We are funded by the <a href='https://ethz.ch/en.html'>ETH Zurich</a> and the
+                <a href='https://www.cdc.gov/'>CDC</a>.
+            </span>
+        </div>
+        <div class='ml-8 basis-1/2 text-balance'>
+            <div class='items-left my-8 flex'>
+                <div class='iconify size-7 mdi--wrench-outline'></div>
+                <h1 class='ml-2 text-3xl text-slate-600'>Open Source Tooling</h1>
+            </div>
+            <span
+                >Code is available on <a href='https://github.com/GenSpectrum'>GitHub</a>, with customizable components
+                to build your own dashboards. We use data from the <a href='https://www.insdc.org/'
                     >International Nucleotide Sequence Database Collaboration (INSDC)</a
-                > and <a href='https://pathoplexus.org/'>Pathoplexus</a>. You can read more about the <a
-                    href={Page.dataSources}>data sources and how we preprocess the data</a
-                >.
-            </li>
-            <li>
-                …'s core team is at the <a href='https://bsse.ethz.ch/cevo'>Computational Evolution Group</a> at ETH Zurich
-                but received valuable contributions from many more. You can find a list of contributors <a
-                    href='https://github.com/GenSpectrum/cov-spectrum-website?tab=readme-ov-file#contributors-'>here</a
-                >.
-            </li>
-            <li>
-                …is supported by funding from <a href='https://ethz.ch/en.html'>ETH Zurich</a>, the <a
-                    href='https://www.cdc.gov/'>U.S. Centers for Disease Control and Prevention</a
-                > and the <a href='https://www.cdc.gov/advanced-molecular-detection/index.html'
-                    >Office of Advanced Molecular Detection</a
-                >.
-            </li>
-            <li>
-                …'s team is very open for feedback and would love to hear from you: you can reach us on <a
-                    href='https://github.com/GenSpectrum/dashboards/issues'
                 >
-                    GitHub</a
+                and <a href='https://pathoplexus.org/'>Pathoplexus</a>, read more about our data processing <a
+                    href={Page.dataSources}>here</a
                 >.
-            </li>
-        </ul>
+            </span>
+        </div>
     </div>
 
-    <h1>Pathogens</h1>
-    <div class='mx-8 mb-8 flex flex-wrap justify-center gap-x-4 gap-y-8'>
+    <div class='mx-auto mb-8 mt-8 flex max-w-[1200px] items-center justify-center'>
+        <div class='iconify size-7 mdi--virus-outline'></div>
```

Comment:
Ah now I see :D You took this icon an colored it for our logo :+1:

Reply:
Yup - I took the svg and added the colors in adobe illustrator (the icons are also available for commercial use which is nice)